package.json

```diff
@@ -26,8 +26,8 @@
     "stage": "NODE_ENV=staging npm run build && publisssh ./dist zooniverse-static/preview.zooniverse.org/pfe-lab"
   },
   "engines": {
-    "node": "~7.10.0",
-    "npm": "~4.2.0"
+    "node": "^8.0.0",
```

Comment:
The latest node and npm are 8.1.2 and 5.0.3 respectively, so we might as well bump to that. Let's also use tilde prefixes to stay consistent with PFE.